app/src/pages/Editor/ExportDocumentDialog.tsx

```diff
@@ -41,24 +41,45 @@ export function ExportDocumentDialog(): JSX.Element {
     });
   });
 
+  const popupState = useSelector((state: RootState) => state.editor.present?.exportPopup);
+
+  const twoFirstSelectionWords = useSelector((state: RootState) => {
+    const result = [];
+    const editorState = state.editor.present;
+    if (!editorState || popupState != 'selection') return '';
+    console.log(selectedItems(editorState));
+    for (const item of selectedItems(editorState)) {
+      if (result.length >= 2) return '_' + result.join('_');
+      if (item.type == 'word') {
+        result.push(item.word);
+        console.log(item.word);
+      }
+    }
+
+    return '';
+  });
+
   const [formState, setFormState] = useState({
     ...exportValues[0],
-    path: documentBasePath + exportValues[0].defaultExtension,
+    path: documentBasePath + twoFirstSelectionWords + exportValues[0].defaultExtension,
```

Comment:
This does not work on my machine (the default filenames do not include the first two words)